cprover_bindings/src/irep/to_irep.rs

```diff
@@ -430,6 +430,10 @@ impl ToIrep for StmtBody {
                     code_irep(IrepId::Decl, vec![lhs.to_irep(mm)])
                 }
             }
+            StmtBody::Deinit(place) => {
+                code_irep(IrepId::Assign, vec![place.to_irep(mm), place.typ().nondet().to_irep(mm)])
```

Comment:
In this spot I would leave a comment explaining "why" this is the right translation.

```
// CBMC doesn't yet have a notion of poison (link?)
// So we translate identically to `nondet` here, but add a comment noting we wish it were poison
// potentially for other backends to pick up and treat specially.
```

Reply:
Yes